prow/github/types.go

```diff
@@ -912,7 +923,11 @@ type ReviewComment struct {
 	UpdatedAt time.Time `json:"updated_at"`
 	// Position will be nil if the code has changed such that the comment is no
 	// longer relevant.
-	Position *int `json:"position"`
+	Position  *int     `json:"position"`
```

Comment:
Can we omitempty these when they are unset?

Reply:
Fixed it. Thanks.